docs/source/rest_api/endpoint_specs.rst

```diff
@@ -0,0 +1,10 @@
+***********************
+Endpoint Specifications
+***********************
+
+These specifications document the available Sawtooth REST API endpoints. They are generated from the official `OpenAPI <http://swagger.io/specification/>`_ formatted YAML specification, which can be found on the `Sawtooth Lake Github <https://github.com/hyperledger/sawtooth-core/blob/master/rest_api/sawtooth_rest_api/openapi.yaml>`_.
+
+.. note::
+   This specification has not been fully implemented. Some query parameters may not function or behave as specified.
```

Comment:
I can manually add a note. I haven't in the past because I didn't want to maintain it every time a change was made. At this point it's only a few query params though, so I suppose it isn't too onerous.